hack/e2e/eventing/peerauthentications/peerauthentications_test.go

```diff
@@ -0,0 +1,133 @@
+//go:build e2e
+// +build e2e
+
+package peerauthentications
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	istiosecv1beta1 "istio.io/api/security/v1beta1"
+	istio "istio.io/client-go/pkg/apis/security/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/ptr"
+
+	. "github.com/kyma-project/eventing-manager/hack/e2e/common"
+	. "github.com/kyma-project/eventing-manager/hack/e2e/common/fixtures"
+	"github.com/kyma-project/eventing-manager/hack/e2e/common/testenvironment"
+	"github.com/stretchr/testify/require"
+)
+
+var testEnvironment *testenvironment.TestEnvironment
+
+// TestMain runs before all the other test functions.
+func TestMain(m *testing.M) {
+	testEnvironment = testenvironment.NewTestEnvironment()
```

Comment:
nit: 
```
testEnvironment = testenvironment.NewTestEnvironment()
```
its hard to immediately spot that one TestEnvironment has a capital `E` whereas the other is a package with lower case `e`
```suggestion
testEnv = testenvironment.New()
```

Reply:
I have renamed to `testEnv`, but I would change the `NewTestEnvironment` in a separate PR because its used in multiple files.